set.mm

```diff
@@ -801861,525 +801980,803 @@ Graph theory (extension)
-      RUUTUVDXSXPXCUUTUUQXPXRVBUUPXCXEVCWSXAXNXOXBVDVEHGABCEFUAJUBDVFVIYOUUSQZY
-      PUUTUULUVDWSXNYOUUSVGUVEUUKUVCJWSUVEUUAUVBUUJUVEYTUVAXMYQYOUUSVJVHVKVLUNV
-      MVNVOVNVOVPVQVRVRVIYMUUHQZYNUUIUUEUUNWNXJYMUUHVGUVFUUDUUMIUVFUUCUULYPUVFU
-      UBUUKJWSUVFYSUUJUUAYMUUHYRVSVTVLWAWBUNVMWKYCUUGUJZXHYAWHWJUVGWIACUCIJWRXM
-      WNXJKDYDYHYFYIUKWCWDWEVNVOVNVOWFWG $.
-  $}
-
-  ${
-    strisomgrop.g $e |- G = <. V , E >. $.
-    strisomgrop.h $e |- H = { <. ( Base ` ndx ) , V >. ,
+      ( cuspgr wcel wa cv wex cpr cfv cgric wbr cgrim co wf1o wb wral c0 brgric
+      wne n0 bitri a1i isuspgrim exbidv bitrd ) ANOBNOPZABUAUBZCQZABUCUDZOZCRZF
+      GUSUEHQZIQZSDOVCUSTVDUSTSEOUFIFUGHFUGPZCRURVBUFUQURUTUHUJVBABUICUTUKULUMU
+      QVAVECHIEDUSABFGJKLMUNUOUP $.
+  $}
+
+  $( The "is isomorphic to" relation for graphs is a relation.  (Contributed by
+     AV, 11-Nov-2022.)  (Revised by AV, 5-May-2025.) $)
+  gricrel $p |- Rel ~=gr $=
+    ( cgric cvv cxp wss wrel cgrim ccnv cdif cima df-gric cnvimass grimfn fndmi
+    c1o cdm sseqtri eqsstri relxp relss mp2 ) ABBCZDUAEAEAFGBNHZIZUAJUCFOUAFUBK
+    UAFLMPQBBRAUAST $.
+
+  $( Graph isomorphism is reflexive for hypergraphs.  (Contributed by AV,
+     11-Nov-2022.)  (Revised by AV, 29-Apr-2025.) $)
+  gricref $p |- ( G e. UHGraph -> G ~=gr G ) $=
+    ( cuhgr wcel cid cvtx cfv cres cgrim co cgric wbr grimid brgrici syl ) ABCD
+    AEFGZAAHICAAJKALAAOMN $.
+
+  ${
+    $d G f $.  $d S f $.
+    $( Graph isomorphism is symmetric for hypergraphs.  (Contributed by AV,
+       11-Nov-2022.)  (Revised by AV, 3-May-2025.) $)
+    gricsym $p |- ( G e. UHGraph -> ( G ~=gr S -> S ~=gr G ) ) $=
+      ( vf cgric wbr cv cgrim co wcel wex cuhgr c0 brgric n0 bitri ccnv grimcnv
+      wne brgrici syl6 exlimdv biimtrid ) BADEZCFZBAGHZIZCJZBKIZABDEZUCUELRUGBA
+      MCUENOUHUFUICUHUFUDPZABGHIUIBAUDQABUJSTUAUB $.
+  $}
+
+  $( Graph isomorphism is symmetric in both directions for hypergraphs.
+     (Contributed by AV, 11-Nov-2022.)  (Proof shortened by AV, 3-May-2025.) $)
+  gricsymb $p |- ( ( A e. UHGraph /\ B e. UHGraph )
+                  -> ( A ~=gr B <-> B ~=gr A ) ) $=
+    ( cuhgr wcel cgric wbr gricsym anbiim ) ACDBCDABEFBAEFBAGABGH $.
+
+  ${
+    $d R f g $.  $d S f g $.  $d T f g $.
+    $( Graph isomorphism is transitive.  (Contributed by AV, 5-Dec-2022.)
+       (Revised by AV, 3-May-2025.) $)
+    grictr $p |- ( ( R ~=gr S /\ S ~=gr T ) -> R ~=gr T ) $=
+      ( vg vf cgric wbr cgrim co c0 wne brgric cv wcel n0 exdistrv ccom syl2anb
+      wex wa grimco ancoms brgrici syl exlimivv sylbir ) ABFGABHIZJKZBCHIZJKZAC
+      FGZBCFGABLBCLUHDMZUGNZDSZEMZUINZESZUKUJDUGOEUIOUNUQTUMUPTZESDSUKUMUPDEPUR
+      UKDEURUOULQZACHINZUKUPUMUTABCUOULUAUBACUSUCUDUEUFRR $.
+  $}
+
+  ${
+    $d g h k $.
+    $( Isomorphism is an equivalence relation on graphs.  (Contributed by AV,
+       3-May-2025.) $)
+    gricer $p |- ( ~=gr i^i ( UHGraph X. UHGraph ) ) Er UHGraph $=
```

Comment:
would this be just hypergraphs instead of graphs?

Reply:
Yes, hypergraphs is more precise here.